x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/overview/license_modal.tsx

```diff
@@ -34,10 +34,16 @@ export const LicenseModal: React.FunctionComponent<Props> = ({
 }) => {
   const { notifications } = useStartServices();
 
-  const { data: licenseText, error: licenseError } = useGetFileByPathQuery(licensePath);
+  const {
+    data: licenseResponse,
+    error: licenseError,
+    isLoading,
+  } = useGetFileByPathQuery(licensePath);
+  const licenseText = licenseResponse?.data;
+  const error = licenseError || licenseResponse?.error;
```

Comment:
Hmm we shouldn't have to check for `licenseResponse?.error` here as `sendRequestForRq` will detect `response.error` and `throw`: https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/public/hooks/use_request/use_request.ts#L57-L59, which will be surfaced as the top-level `error` property on this query result object.

Reply:
Thanks for pointing it out Kyle, I'll remove it before merging